api/providers.go

```diff
@@ -83,6 +92,11 @@ func loadExternalProviders(endpointsConfgFiles []string) (externalProviders LogP
 }
 
 func loadSystemdCollectors(cfg *config.Config, DCOSTools dcos.Tooler) ([]collector.Collector, error) {
+	if runtime.GOOS == GoosWindows || runtime.GOOS == GoosDarwin {
+		logrus.Debug("Ignoring Systemd collectors on Windows.")
+		return make([]collector.Collector, 0), nil
```

Comment:
```suggestion
		return nil, nil
```

Reply:
Are you sure `nil` is correct? The collectors should be an empty list.

Reply:
A nil slice is basically an empty slice in Go (https://tour.golang.org/moretypes/12). Welcome to Go land. 😉